frontend-react/src/webreceiver-utils.ts

```diff
@@ -26,9 +26,7 @@ const permissionCheck = (permission: string, authState: AuthState) => {
     if (permission === PERMISSIONS.RECEIVER) {
         return reportReceiver(authState);
     }
-    return authState.accessToken?.claims.organization.find((o: string) =>
-        o.includes(permission)
-    );
+    return authState.accessToken?.claims.organization.includes(permission);
```

Comment:
What do you think about a `try {} catch(e) {console.error(e);}` around this function... just for safety?